samples/request-cancellation/worker.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2017-2023 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+export default {
+  async fetch(request) {
+    request.signal.addEventListener('abort', () => {
+      console.log('Request was aborted');
+    });
+
+    console.log('Starting long-running task');
+    // Simulate a long-running task so we can test aborting
+    await new Promise((resolve) => setTimeout(resolve, 3000));
```

Comment:
Try also adding:

```
    ctx.waitUntil(new Promise((resolve) => setTimeout(resolve, 4000)));
```

Without the waitUntil(), as soon as the client disconnects, the entire request context is torn down immediately, and hence the abort event doesn't get a chance to be delivered.

Reply:
Just gave that a try in https://github.com/cloudflare/workerd/pull/5062/commits/07ecdbd99171dd3f171cc8d62111dff9012056fd, but I'm still not seeing any logs.

Reply:
Doesn't seem to work locally for me either. I'll try putting a debugger on it, unless anyone has any leads.